src/jqLite.js

```diff
@@ -90,6 +90,16 @@
  * - [`val()`](http://api.jquery.com/val/)
  * - [`wrap()`](http://api.jquery.com/wrap/)
  *
+ * jqLite also provides a method restoring pre-1.8 insecure treatment of XHTML-like tags
+ * that makes input like `<div /><span />` turned to `<div></div><span></span>` instead of
+ * `<div><span></span></div>` like version 1.8 & newer do:
```

Comment:
@Splaktar Would this work better?
```suggestion
 * jqLite also provides a method restoring pre-1.8 insecure treatment of XHTML-like tags.
 * This legacy behavior turns input like `<div /><span />` to `<div></div><span></span>`
 * instead of `<div><span></span></div>` like version 1.8 & newer do:
```

Reply:
Or:
```suggestion
 * jqLite also provides a method restoring pre-1.8 insecure treatment of XHTML-like tags.
 * This legacy behavior turns input like `<div /><span />` to `<div></div><span></span>`
 * instead of `<div><span></span></div>` like version 1.8 & newer do. To restore it, invoke:
```
to provide context for the call below.

Reply:
```suggestion
 * jqLite provides a method for restoring pre-1.8 insecure treatment of XHTML-like tags.
 * This legacy behavior turns input like `<div /><span />` into `<div></div><span></span>`
 * instead of `<div><span></span></div>` like version 1.8 & newer do. To restore the legacy behavior, invoke:
```

Reply:
Alternatively, this could work
```
 * This legacy behavior converts input like `<div /><span />` to `<div></div><span></span>`
```